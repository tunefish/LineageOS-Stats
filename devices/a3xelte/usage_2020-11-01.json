{
  "15.1-20190125-unofficial-a3xelte": 9,
  "15.1-20190212-unofficial-a3xelte": 4,
  "15.1-20190310-unofficial-a3xelte": 8,
  "15.1-20190408-unofficial-a3xelte": 12,
  "15.1-20190516-unofficial-a3xelte": 3,
  "15.1-20190623-unofficial-a3xelte": 7,
  "15.1-20190709-unofficial-a3xelte": 23,
  "15.1-20190923-dnjl-a3xelte": 1,
  "16.0-20190326-unofficial-a3xelte": 2,
  "16.0-20190408-unofficial-a3xelte": 9,
  "16.0-20190517-unofficial-a3xelte": 2,
  "16.0-20190521-unofficial-a3xelte": 1,
  "16.0-20190523-unofficial-a3xelte": 1,
  "16.0-20190526-unofficial-a3xelte": 5,
  "16.0-20190719-unofficial-a3xelte": 2,
  "16.0-20190723-unofficial-a3xelte": 6,
  "16.0-20190811-unofficial-a3xelte": 2,
  "16.0-20190816-unofficial-a3xelte": 1,
  "16.0-20190818-unofficial-a3xelte": 13,
  "16.0-20190826-nightly-a3xelte": 2,
  "16.0-20190902-nightly-a3xelte": 1,
  "16.0-20190903-nightly-a3xelte": 1,
  "16.0-20190905-nightly-a3xelte": 1,
  "16.0-20190906-nightly-a3xelte": 1,
  "16.0-20190908-nightly-a3xelte": 1,
  "16.0-20190912-nightly-a3xelte": 1,
  "16.0-20190914-nightly-a3xelte": 2,
  "16.0-20190921-nightly-a3xelte": 2,
  "16.0-20190924-nightly-a3xelte": 1,
  "16.0-20190928-nightly-a3xelte": 2,
  "16.0-20190929-nightly-a3xelte": 1,
  "16.0-20190930-nightly-a3xelte": 2,
  "16.0-20191002-nightly-a3xelte": 1,
  "16.0-20191004-nightly-a3xelte": 3,
  "16.0-20191007-microg-a3xelte": 1,
  "16.0-20191012-nightly-a3xelte": 1,
  "16.0-20191013-nightly-a3xelte": 1,
  "16.0-20191014-microg-a3xelte": 1,
  "16.0-20191016-nightly-a3xelte": 1,
  "16.0-20191020-nightly-a3xelte": 1,
  "16.0-20191021-microg-a3xelte": 1,
  "16.0-20191023-nightly-a3xelte": 3,
  "16.0-20191025-nightly-a3xelte": 2,
  "16.0-20191027-nightly-a3xelte": 3,
  "16.0-20191101-nightly-a3xelte": 3,
  "16.0-20191105-nightly-a3xelte": 1,
  "16.0-20191107-unofficial-a3xelte": 2,
  "16.0-20191109-nightly-a3xelte": 1,
  "16.0-20191110-nightly-a3xelte": 1,
  "16.0-20191112-nightly-a3xelte": 1,
  "16.0-20191113-nightly-a3xelte": 1,
  "16.0-20191116-nightly-a3xelte": 2,
  "16.0-20191118-nightly-a3xelte": 1,
  "16.0-20191122-nightly-a3xelte": 2,
  "16.0-20191129-nightly-a3xelte": 1,
  "16.0-20191130-nightly-a3xelte": 1,
  "16.0-20191201-nightly-a3xelte": 1,
  "16.0-20191202-microg-a3xelte": 1,
  "16.0-20191202-nightly-a3xelte": 1,
  "16.0-20191203-nightly-a3xelte": 3,
  "16.0-20191204-nightly-a3xelte": 2,
  "16.0-20191210-nightly-a3xelte": 1,
  "16.0-20191211-nightly-a3xelte": 1,
  "16.0-20191213-nightly-a3xelte": 2,
  "16.0-20191217-nightly-a3xelte": 1,
  "16.0-20191221-nightly-a3xelte": 1,
  "16.0-20191222-nightly-a3xelte": 2,
  "16.0-20191223-nightly-a3xelte": 3,
  "16.0-20191228-nightly-a3xelte": 3,
  "16.0-20191231-nightly-a3xelte": 1,
  "16.0-20200101-nightly-a3xelte": 2,
  "16.0-20200104-nightly-a3xelte": 3,
  "16.0-20200110-nightly-a3xelte": 2,
  "16.0-20200111-nightly-a3xelte": 1,
  "16.0-20200112-nightly-a3xelte": 2,
  "16.0-20200114-nightly-a3xelte": 3,
  "16.0-20200115-nightly-a3xelte": 3,
  "16.0-20200116-nightly-a3xelte": 2,
  "16.0-20200117-nightly-a3xelte": 1,
  "16.0-20200118-nightly-a3xelte": 1,
  "16.0-20200119-nightly-a3xelte": 1,
  "16.0-20200126-nightly-a3xelte": 2,
  "16.0-20200129-nightly-a3xelte": 1,
  "16.0-20200130-nightly-a3xelte": 1,
  "16.0-20200131-nightly-a3xelte": 4,
  "16.0-20200201-nightly-a3xelte": 4,
  "16.0-20200204-nightly-a3xelte": 1,
  "16.0-20200206-nightly-a3xelte": 1,
  "16.0-20200208-nightly-a3xelte": 1,
  "16.0-20200209-nightly-a3xelte": 2,
  "16.0-20200210-microg-a3xelte": 1,
  "16.0-20200210-nightly-a3xelte": 1,
  "16.0-20200211-nightly-a3xelte": 1,
  "16.0-20200212-nightly-a3xelte": 2,
  "16.0-20200214-nightly-a3xelte": 3,
  "16.0-20200215-nightly-a3xelte": 3,
  "16.0-20200218-nightly-a3xelte": 1,
  "16.0-20200219-nightly-a3xelte": 2,
  "16.0-20200221-nightly-a3xelte": 3,
  "16.0-20200223-nightly-a3xelte": 3,
  "16.0-20200224-microg-a3xelte": 1,
  "16.0-20200225-nightly-a3xelte": 1,
  "16.0-20200229-nightly-a3xelte": 4,
  "16.0-20200304-nightly-a3xelte": 1,
  "16.0-20200305-nightly-a3xelte": 3,
  "16.0-20200308-nightly-a3xelte": 1,
  "16.0-20200310-nightly-a3xelte": 2,
  "16.0-20200311-nightly-a3xelte": 2,
  "16.0-20200313-nightly-a3xelte": 2,
  "16.0-20200315-nightly-a3xelte": 2,
  "16.0-20200316-nightly-a3xelte": 1,
  "16.0-20200317-nightly-a3xelte": 2,
  "16.0-20200319-nightly-a3xelte": 2,
  "16.0-20200320-nightly-a3xelte": 1,
  "16.0-20200321-nightly-a3xelte": 1,
  "16.0-20200322-nightly-a3xelte": 2,
  "16.0-20200324-nightly-a3xelte": 2,
  "16.0-20200325-nightly-a3xelte": 16,
  "16.0-20200412-nightly-a3xelte": 14,
  "16.0-20200419-nightly-a3xelte": 143,
  "16.0-20200427-nightly-a3xelte": 17,
  "16.0-20200428-unofficial-a3xelte": 1,
  "16.0-20200507-microg-a3xelte": 2,
  "16.0-20200514-microg-a3xelte": 2,
  "16.0-20200521-microg-a3xelte": 5,
  "16.0-20200611-unofficial-a3xelte": 16,
  "17.1-20200117-unofficial-a3xelte": 7,
  "17.1-20200121-unofficial-a3xelte": 1,
  "17.1-20200210-unofficial-a3xelte": 2,
  "17.1-20200216-unofficial-a3xelte": 1,
  "17.1-20200307-unofficial-a3xelte": 2,
  "17.1-20200312-unofficial-a3xelte": 9,
  "17.1-20200412-unofficial-a3xelte": 12,
  "17.1-20200516-unofficial-a3xelte": 108,
  "17.1-20200529-nightly-a3xelte": 1,
  "17.1-20200530-nightly-a3xelte": 3,
  "17.1-20200601-nightly-a3xelte": 5,
  "17.1-20200604-nightly-a3xelte": 1,
  "17.1-20200607-nightly-a3xelte": 8,
  "17.1-20200614-nightly-a3xelte": 4,
  "17.1-20200621-nightly-a3xelte": 17,
  "17.1-20200628-nightly-a3xelte": 17,
  "17.1-20200712-nightly-a3xelte": 9,
  "17.1-20200715-unofficial-tissot": 9,
  "17.1-20200716-unofficial-herolte": 6,
  "17.1-20200719-nightly-a3xelte": 10,
  "17.1-20200726-nightly-a3xelte": 24,
  "17.1-20200809-nightly-a3xelte": 37,
  "17.1-20200816-nightly-a3xelte": 44,
  "17.1-20200818-microg-a3xelte": 3,
  "17.1-20200821-unofficial-z3": 25,
  "17.1-20200824-microg-a3xelte": 7,
  "17.1-20200830-nightly-a3xelte": 16,
  "17.1-20200830-unofficial-tissot": 1,
  "17.1-20200906-nightly-a3xelte": 38,
  "17.1-20200913-nightly-a3xelte": 43,
  "17.1-20200920-nightly-a3xelte": 138,
  "17.1-20200921-microg-a3xelte": 13,
  "17.1-20200927-nightly-a3xelte": 159,
  "17.1-20201004-nightly-a3xelte": 137,
  "17.1-20201011-nightly-a3xelte": 183,
  "17.1-20201012-microg-a3xelte": 15,
  "17.1-20201018-nightly-a3xelte": 170,
  "17.1-20201018-unofficial-a3xelte": 1,
  "17.1-20201025-nightly-a3xelte": 235,
  "nrd90m.a310fxxu3cqe6": 1
}