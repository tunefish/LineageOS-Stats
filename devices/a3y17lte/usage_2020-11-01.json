{
  "14.1-20180205-unofficial-a3y17lte": 1,
  "14.1-20180614-unofficial-j5xnlte": 5216,
  "15.1-20180306-unofficial-a3y17lte": 1,
  "15.1-20180517-unofficial-a3y17lte": 5,
  "15.1-20180821-unofficial-a3y17lte": 4,
  "15.1-20181205-unofficial-a3y17lte": 118,
  "16.0-20181215-unofficial-a3y17lte": 15,
  "16.0-20190307-unofficial-a3y17lte": 8,
  "16.0-20190330-unofficial-a3y17lte": 7,
  "16.0-20190410-unofficial-a3y17lte": 7,
  "16.0-20190419-unofficial-a3y17lte": 1,
  "16.0-20190426-unofficial-a3y17lte": 3,
  "16.0-20190430-unofficial-a3y17lte": 2,
  "16.0-20190507-unofficial-a3y17lte": 1,
  "16.0-20190510-unofficial-a3y17lte": 2,
  "16.0-20190517-unofficial-a3y17lte": 3,
  "16.0-20190522-unofficial-a3y17lte": 7,
  "16.0-20190529-unofficial-a3y17lte": 2,
  "16.0-20190611-unofficial-a3y17lte": 2,
  "16.0-20190623-unofficial-a3y17lte": 49,
  "16.0-20190824-unofficial-a3y17lte": 1,
  "16.0-20190908-unofficial-a3y17lte": 302,
  "16.0-20190926-unofficial-a3y17lte": 2,
  "16.0-20191008-unofficial-a3y17lte": 42,
  "16.0-20191126-unofficial-a3y17lte": 3,
  "16.0-20200621-unofficial-a3y17lte": 7,
  "17.1-20200119-unofficial-a3y17lte": 1,
  "17.1-20200125-unofficial-a3y17lte": 1,
  "17.1-20200210-unofficial-a3y17lte": 1,
  "17.1-20200221-unofficial-a3y17lte": 9,
  "17.1-20200403-unofficial-a3y17lte": 21,
  "17.1-20200613-unofficial-a3y17lte": 10,
  "17.1-20200713-unofficial-a3y17lte": 26,
  "17.1-20200715-unofficial-tissot": 6,
  "17.1-20200716-unofficial-herolte": 1,
  "17.1-20200821-unofficial-z3": 13,
  "17.1-20200825-unofficial-a3y17lte": 5,
  "17.1-20200830-unofficial-a3y17lte": 302,
  "17.1-20201009-unofficial-a3y17lte": 1,
  "17.1-20201016-unofficial-a3y17lte": 1,
  "18.0-20201028-unofficial-a3y17lte": 1
}