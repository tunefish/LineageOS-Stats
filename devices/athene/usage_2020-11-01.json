{
  "1.0.1-athene": 2,
  "13.0-20170519-unofficial-athene": 6,
  "13.0-20170816-unofficial-athene": 27,
  "13.0-20170915-unofficial-athene": 3,
  "13.0-20180930-unofficial-athene": 5,
  "14.1-20170113-unofficial-athene": 2,
  "14.1-20170123-nightly-athene": 2,
  "14.1-20170124-unofficial-athene": 1,
  "14.1-20170130-nightly-athene": 1,
  "14.1-20170205-unofficial-athene": 2,
  "14.1-20170206-nightly-athene": 2,
  "14.1-20170213-nightly-athene": 7,
  "14.1-20170227-nightly-athene": 1,
  "14.1-20170303-unofficial-athene": 1,
  "14.1-20170306-nightly-athene": 1,
  "14.1-20170311-unofficial-athene": 8,
  "14.1-20170313-nightly-athene": 1,
  "14.1-20170320-nightly-athene": 4,
  "14.1-20170323-unofficial-athene": 3,
  "14.1-20170327-nightly-athene": 3,
  "14.1-20170403-nightly-athene": 1,
  "14.1-20170410-nightly-athene": 9,
  "14.1-20170424-nightly-athene": 2,
  "14.1-20170501-nightly-athene": 4,
  "14.1-20170503-unofficial-athene": 1,
  "14.1-20170515-nightly-athene": 1,
  "14.1-20170517-unofficial-athene": 2,
  "14.1-20170522-nightly-athene": 26,
  "14.1-20170612-nightly-athene": 2,
  "14.1-20170712-legendrom-17.0-athene": 2,
  "14.1-20170716-legendrom-17.0-athene": 1,
  "14.1-20170717-nightly-athene": 7,
  "14.1-20170724-nightly-athene": 5,
  "14.1-20170731-nightly-athene": 7,
  "14.1-20170807-nightly-athene": 8,
  "14.1-20170814-nightly-athene": 2,
  "14.1-20170821-nightly-athene": 5,
  "14.1-20170828-nightly-athene": 5,
  "14.1-20170904-nightly-athene": 8,
  "14.1-20170911-nightly-athene": 6,
  "14.1-20170918-nightly-athene": 6,
  "14.1-20170925-nightly-athene": 4,
  "14.1-20171002-nightly-athene": 6,
  "14.1-20171009-nightly-athene": 6,
  "14.1-20171016-nightly-athene": 3,
  "14.1-20171023-nightly-athene": 10,
  "14.1-20171030-nightly-athene": 3,
  "14.1-20171106-nightly-athene": 10,
  "14.1-20171113-nightly-athene": 3,
  "14.1-20171120-nightly-athene": 9,
  "14.1-20171127-nightly-athene": 10,
  "14.1-20171204-nightly-athene": 13,
  "14.1-20171211-nightly-athene": 6,
  "14.1-20171218-nightly-athene": 5,
  "14.1-20171225-nightly-athene": 8,
  "14.1-20180101-nightly-athene": 7,
  "14.1-20180108-nightly-athene": 14,
  "14.1-20180122-nightly-athene": 9,
  "14.1-20180129-nightly-athene": 12,
  "14.1-20180205-nightly-athene": 10,
  "14.1-20180212-nightly-athene": 14,
  "14.1-20180219-nightly-athene": 10,
  "14.1-20180228-nightly-athene": 7,
  "14.1-20180307-nightly-athene": 11,
  "14.1-20180314-nightly-athene": 9,
  "14.1-20180321-nightly-athene": 9,
  "14.1-20180328-nightly-athene": 18,
  "14.1-20180404-nightly-athene": 2,
  "14.1-20180410-nightly-athene": 16,
  "14.1-20180418-nightly-athene": 14,
  "14.1-20180425-nightly-athene": 9,
  "14.1-20180430-unofficial-athene": 2,
  "14.1-20180502-microg-athene": 3,
  "14.1-20180503-nightly-athene": 13,
  "14.1-20180509-nightly-athene": 12,
  "14.1-20180516-nightly-athene": 16,
  "14.1-20180523-nightly-athene": 16,
  "14.1-20180530-microg-athene": 1,
  "14.1-20180530-nightly-athene": 14,
  "14.1-20180606-nightly-athene": 18,
  "14.1-20180613-nightly-athene": 17,
  "14.1-20180620-nightly-athene": 24,
  "14.1-20180627-nightly-athene": 13,
  "14.1-20180704-nightly-athene": 23,
  "14.1-20180711-nightly-athene": 15,
  "14.1-20180718-nightly-athene": 16,
  "14.1-20180725-nightly-athene": 17,
  "14.1-20180801-nightly-athene": 12,
  "14.1-20180805-unofficial-athene": 2,
  "14.1-20180808-nightly-athene": 18,
  "14.1-20180810-official-athene": 1,
  "14.1-20180815-nightly-athene": 27,
  "14.1-20180822-nightly-athene": 14,
  "14.1-20180829-nightly-athene": 13,
  "14.1-20180905-nightly-athene": 24,
  "14.1-20180912-microg-athene": 1,
  "14.1-20180912-nightly-athene": 27,
  "14.1-20180919-nightly-athene": 21,
  "14.1-20180926-nightly-athene": 26,
  "14.1-20181003-nightly-athene": 31,
  "14.1-20181010-nightly-athene": 27,
  "14.1-20181017-nightly-athene": 29,
  "14.1-20181017-official-athene": 2,
  "14.1-20181024-nightly-athene": 34,
  "14.1-20181031-nightly-athene": 62,
  "14.1-20181107-nightly-athene": 510,
  "14.1-20181121-microg-athene": 19,
  "14.1-20181206-unofficial-athene": 1,
  "14.1-20181207-unofficial-athene": 9,
  "14.1-20181208-unofficial-athene": 6,
  "14.1-20190131-unofficial-athene": 19,
  "14.1-20190407-unofficial-athene": 17,
  "14.1-20190507-unofficial-athene": 10,
  "14.1-20190517-unofficial-athene": 1,
  "14.1-20190530-unofficial-athene": 1,
  "14.1-20190608-unofficial-athene": 8,
  "14.1-20190708-unofficial-athene": 9,
  "14.1-20190723-official-athene": 3,
  "14.1-20190807-unofficial-athene": 4,
  "14.1-20190906-official-athene": 6,
  "14.1-20190907-unofficial-athene": 292,
  "14.1-20191223-unofficial-athene": 2,
  "14.1-20200130-official-athene": 5,
  "14.1-20200214-unofficial-athene": 2,
  "14.1-20200330-unofficial-athene": 1,
  "14.1-20200404-unofficial-athene": 10,
  "14.1-20200519-official-athene": 14,
  "14.1-20200610-unofficial-athene": 1,
  "14.1-20200613-unofficial-athene": 1,
  "14.1-20200704-unofficial-athene": 2,
  "14.1-20200807-unofficial-athene": 4,
  "14.1-20200809-unofficial-athene": 1,
  "14.1-20200822-unofficial-athene": 12,
  "14.1-20201006-unofficial-athene": 8,
  "14.1-20201017-unofficial-athene": 1,
  "14.1-20201027-unofficial-athene": 5,
  "15.1-20180203-unofficial-athene": 2,
  "15.1-20180318-unofficial-athene": 16,
  "15.1-20180325-unofficial-athene": 3,
  "15.1-20180517-unofficial-athene": 40,
  "15.1-20180526-unofficial-athene": 1,
  "15.1-20180615-unofficial-athene": 2,
  "15.1-20180621-unofficial-athene": 3,
  "15.1-20180714-unofficial-athene": 7,
  "15.1-20180721-unofficial-athene": 1,
  "15.1-20180805-unofficial-athene": 4,
  "15.1-20180930-unofficial-athene": 9,
  "15.1-20181007-unofficial-athene": 3,
  "15.1-20181107-unofficial-athene": 15,
  "15.1-20190223-unofficial-athene": 6,
  "15.1-20190403-unofficial-athene": 2,
  "15.1-20190413-unofficial-athene": 3,
  "15.1-20190506-unofficial-athene": 1,
  "15.1-20190509-unofficial-athene": 4,
  "15.1-20190518-unofficial-athene": 8,
  "15.1-20190616-unofficial-athene": 2,
  "15.1-20190624-unofficial-athene": 3,
  "15.1-20190630-unofficial-athene": 4,
  "15.1-20190704-unofficial-athene": 3,
  "15.1-20190804-unofficial-athene": 3,
  "15.1-20190808-unofficial-athene": 4,
  "15.1-20190825-unofficial-athene": 7,
  "15.1-20190907-unofficial-athene": 13,
  "15.1-20191011-unofficial-athene": 30,
  "15.1-20191215-unofficial-athene": 1,
  "15.1-20191217-unofficial-athene": 7,
  "15.1-20200105-unofficial-athene": 5,
  "15.1-20200108-unofficial-athene": 67,
  "15.1-20200124-unofficial-athene": 3,
  "15.1-20200211-unofficial-athene": 8,
  "15.1-20200309-unofficial-athene": 2,
  "15.1-20200407-unofficial-athene": 51,
  "15.1-20200424-unofficial-athene": 1,
  "15.1-20200425-unofficial-athene": 8,
  "15.1-20200509-unofficial-athene": 1,
  "15.1-20200516-unofficial-athene": 1,
  "15.1-20200527-unofficial-athene": 39,
  "15.1-20200609-unofficial-athene": 4,
  "15.1-20200723-unofficial-athene": 1,
  "15.1-20200807-unofficial-athene": 3,
  "15.1-20200816-unofficial-athene": 33,
  "15.1-20200822-unofficial-athene": 16,
  "15.1-20200920-unofficial-athene": 282,
  "15.1-20200925-unofficial-athene": 10,
  "15.1-20201011-unofficial-athene": 17,
  "15.1-20201012-unofficial-athene": 7,
  "15.1-20201025-unofficial-athene": 6,
  "15.1-20201026-unofficial-athene": 9,
  "16.0-20181030-unofficial-athene": 3,
  "16.0-20181130-unofficial-athene": 1,
  "16.0-20181223-unofficial-athene": 2,
  "16.0-20190116-unofficial-athene": 106,
  "16.0-20200508-unofficial-athene": 1,
  "16.0-20200511-unofficial-athene": 3,
  "16.0-20200613-unofficial-athene": 3,
  "16.0-20200614-unofficial-athene": 10,
  "16.0-20200724-unofficial-athene": 10,
  "16.0-20200809-unofficial-athene": 4,
  "16.0-20200916-unofficial-athene": 54,
  "16.0-20200928-unofficial-athene": 84,
  "16.0-20201014-unofficial-athene": 47,
  "16.0-20201015-unofficial-athene": 77,
  "lineage_athene-userdebug 7.1.1 nof27b 8e19d17742 test-keys": 3,
  "lineage_athene-userdebug 7.1.2 n2g47o 759a28a800 test-keys": 1,
  "lineage_athene-userdebug 7.1.2 njh47f cfee3a73a6 test-keys": 1,
  "lineage_athene-userdebug 9 pq3a.190801.002 eng.deivid.20201014.024404 test-keys": 1,
  "n2g47e": 8,
  "njh47b": 2,
  "njh47d": 8,
  "njh47f": 119,
  "nof27b": 1,
  "nuclearom-v-official[7.1.2]--20170604-0631": 1,
  "nuclearom-v3.0.0-unofficial[7.1.2]--20170628-0920": 1
}