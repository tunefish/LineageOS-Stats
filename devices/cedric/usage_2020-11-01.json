{
  "1.0.1-cedric": 1,
  "14.1-20170601-unofficial-cedric": 1,
  "14.1-20170706-unofficial-cedric": 3,
  "14.1-20170811-unofficial-cedric": 22,
  "14.1-20171006-unofficial-cedric": 3,
  "14.1-20171018-unofficial-cedric": 4,
  "14.1-20171111_140556-unofficial-cedric": 18,
  "14.1-20180117-unofficial-cedric": 25,
  "14.1-20180204-unofficial-cedric": 1,
  "14.1-20180318-unofficial-cedric": 46,
  "14.1-20180817-unofficial-cedric": 45,
  "14.1-20190207-unofficial-cedric": 1,
  "14.1-20190221-unofficial-cedric": 2,
  "14.1-20190311-unofficial-cedric": 1,
  "14.1-20190407-unofficial-cedric": 2,
  "14.1-20190630-unofficial-cedric": 4,
  "14.1-20190706-unofficial-cedric": 1,
  "14.1-20190710-unofficial-cedric": 7,
  "14.1-20190813-unofficial-cedric": 5,
  "14.1-20190912-unofficial-cedric": 20,
  "14.1-20191023-unofficial-cedric": 1,
  "14.1-20191115-unofficial-cedric": 9,
  "14.1-20191228-unofficial-cedric": 1,
  "14.1-20200214-unofficial-cedric": 1,
  "14.1-20200216-unofficial-cedric": 1,
  "14.1-20200304-unofficial-cedric": 1,
  "14.1-20200328-unofficial-cedric": 1,
  "14.1-20200407-unofficial-cedric": 1,
  "14.1-20200424-unofficial-cedric": 3,
  "14.1-20200508-unofficial-cedric": 2,
  "14.1-20200610-unofficial-cedric": 3,
  "14.1-20200708-unofficial-cedric": 2,
  "14.1-20200805-unofficial-cedric": 3,
  "14.1-20200822-unofficial-cedric": 6,
  "14.1-20200910-unofficial-cedric": 1,
  "14.1-20200919-unofficial-cedric": 22,
  "14.1-20201006-unofficial-cedric": 14,
  "15.1-20180213-unofficial-cedric": 21,
  "15.1-20180415-unofficial-cedric": 102,
  "15.1-20180422-unofficial-cedric": 11,
  "15.1-20180424-unofficial-cedric": 2,
  "15.1-20180715-unofficial-cedric": 1,
  "15.1-20180723-unofficial-cedric": 4,
  "15.1-20180809-unofficial-cedric": 1,
  "15.1-20180828-unofficial-cedric": 2,
  "15.1-20181016-unofficial-cedric": 2,
  "15.1-20190228-unofficial-cedric": 1,
  "15.1-20190425-unofficial-cedric": 3,
  "15.1-20200927-unofficial-cedric": 4,
  "15.1-20201003-unofficial-cedric": 17,
  "15.1-20201012-unofficial-cedric": 3,
  "15.1-20201021-unofficial-cedric": 3,
  "15.1-20201024-unofficial-cedric": 2,
  "16.0-20190101-unofficial-cedric": 57,
  "16.0-20190405-unofficial-cedric": 27,
  "16.0-20190522-unofficial-cedric": 26,
  "16.0-20190525-unofficial-cedric": 1,
  "16.0-20190701-unofficial-cedric": 6,
  "16.0-20190709-unofficial-cedric": 41,
  "16.0-20190711-unofficial-cedric": 4,
  "16.0-20190719-unofficial-cedric": 1,
  "16.0-20190817-unofficial-cedric": 2,
  "16.0-20190912-unofficial-cedric": 186,
  "16.0-20191016-unofficial-cedric": 1,
  "16.0-20191211-unofficial-cedric": 1,
  "16.0-20191214-unofficial-cedric": 2,
  "16.0-20200211-unofficial-cedric": 2,
  "16.0-20200311-unofficial-cedric": 3,
  "16.0-20200511-unofficial-cedric": 1,
  "16.0-20200522-unofficial-cedric": 1,
  "16.0-20200819-unofficial-cedric": 1,
  "16.0-20200914-unofficial-cedric": 1,
  "16.0-20200915-unofficial-cedric": 1,
  "16.0-20200919-unofficial-cedric": 2,
  "17.0-20191214-unofficial-cedric": 137,
  "17.1-20200112-unofficial-cedric": 22,
  "17.1-20200220-unofficial-cedric": 38,
  "17.1-20200319-unofficial-cedric": 3,
  "17.1-20200419-unofficial-cedric": 2,
  "17.1-20200422-unofficial-cedric": 106,
  "17.1-20200520-unofficial-cedric": 3,
  "17.1-20200602-unofficial-cedric": 1,
  "17.1-20200612-unofficial-cedric": 169,
  "17.1-20200623-unofficial-cedric": 1,
  "17.1-20200625-unofficial-cedric": 1,
  "17.1-20200709-unofficial-cedric": 1,
  "17.1-20200715-unofficial-cedric": 2,
  "17.1-20200717-unofficial-cedric": 1,
  "17.1-20200724-unofficial-cedric": 1,
  "17.1-20200731-unofficial-cedric": 1,
  "17.1-20200808-unofficial-cedric": 1,
  "17.1-20200810-unofficial-cedric": 10,
  "17.1-20200813-unofficial-cedric": 2,
  "17.1-20200814-unofficial-cedric": 1,
  "17.1-20200815-unofficial-cedric": 2,
  "17.1-20200816-unofficial-cedric": 1,
  "17.1-20200818-unofficial-cedric": 4,
  "17.1-20200819-unofficial-cedric": 1,
  "17.1-20200820-unofficial-cedric": 631,
  "17.1-20200822-unofficial-cedric": 120,
  "17.1-20200903-unofficial-cedric": 1,
  "17.1-20200905-unofficial-cedric": 1,
  "17.1-20200908-unofficial-cedric": 1,
  "17.1-20200909-unofficial-cedric": 1,
  "17.1-20200910-unofficial-cedric": 1,
  "17.1-20200911-unofficial-cedric": 1,
  "17.1-20200913-unofficial-cedric": 2,
  "17.1-20200915-unofficial-cedric": 5,
  "17.1-20200917-unofficial-cedric": 1,
  "17.1-20200919-unofficial-cedric": 1,
  "17.1-20200920-unofficial-cedric": 6,
  "17.1-20200921-unofficial-cedric": 3,
  "17.1-20200922-unofficial-cedric": 44,
  "17.1-20200927-unofficial-cedric": 1,
  "17.1-20201004-unofficial-cedric": 3,
  "17.1-20201006-unofficial-cedric": 2,
  "17.1-20201009-unofficial-cedric": 1,
  "17.1-20201016-unofficial-cedric": 7,
  "17.1-20201018-unofficial-cedric": 8,
  "17.1-20201019-unofficial-cedric": 4,
  "17.1-20201020-unofficial-cedric": 4,
  "17.1-20201021-unofficial-cedric": 12,
  "17.1-20201022-unofficial-cedric": 4,
  "17.1-20201023-unofficial-cedric": 2,
  "17.1-20201024-unofficial-cedric": 8,
  "17.1-20201025-unofficial-cedric": 1,
  "17.1-20201026-unofficial-cedric": 14,
  "17.1-20201027-unofficial-cedric": 14,
  "17.1-20201028-unofficial-cedric": 10,
  "17.1-20201029-unofficial-cedric": 6,
  "17.1-20201030-unofficial-cedric": 5,
  "17.1-20201031-unofficial-cedric": 6,
  "aosp_cedric-userdebug 9 pq3a.190801.002 eng.thxbro.20190813.181708 test-keys": 2,
  "qq3a.200705.002": 10,
  "ro.lineage.releasetype=unofficial": 3,
  "rr_cedric-userdebug 10 qp1a.191105.004 eng.root.20191215.073423 test-keys": 11
}