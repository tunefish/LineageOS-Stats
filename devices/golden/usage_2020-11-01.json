{
  "13.0-20170121-unofficial-golden": 1,
  "13.0-20170129-unofficial-golden": 56,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 5,
  "13.0-20170310-unofficial-golden": 11,
  "13.0-20170328-unofficial-golden": 310,
  "13.0-20170420-unofficial-golden": 25,
  "14.1-20170122-unofficial-golden": 161,
  "14.1-20170318-unofficial-golden": 3,
  "14.1-20170402-unofficial-golden": 3,
  "14.1-20170415-unofficial-golden": 4,
  "14.1-20170419-unofficial-golden": 4,
  "14.1-20170507-unofficial-golden": 15,
  "14.1-20170522-unofficial-golden": 1,
  "14.1-20170609-unofficial-golden": 282,
  "14.1-20170610-unofficial-golden": 5,
  "14.1-20170628-unofficial-golden": 488,
  "14.1-20170724-unofficial-golden": 27,
  "14.1-20170817-unofficial-golden": 17,
  "14.1-20170920-unofficial-golden": 6,
  "14.1-20171011-unofficial-golden": 43,
  "14.1-20171215_192311-official-golden": 66,
  "14.1-20171215_192311-unofficial-golden": 426,
  "14.1-20180109_002536-unofficial-golden": 117,
  "14.1-20181014-unofficial-golden": 1,
  "14.1-20200817_071197-official-golden-rogerwed": 1,
  "15.1-20181030-unofficial-golden": 5,
  "golden-user 8.1.0 oe632e 3de271a2ce release-keys": 1,
  "njh47f release-keys": 1
}